sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareOperationProperties.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// HealthcareOperationProperties.
+    /// </summary>
+    public class HealthcareOperationProperties
```

Comment:
This proposal doesn't follow the design here => https://apiview.dev/Assemblies/Review/b196a5e53fa2441f92e3b1accefa35e7?doc=False#Azure.AI.TextAnalytics.Healthcare.AnalyzeHealthcareEntitiesOperation
where these properties are part of the `HealthcareOperation` class.

Could you elaborate more on why the preference to keep them separated?

Reply:
I was trying to follow KV design, i can remove it.